packages/api-clients/open-api/eserviceTemplateApi.yml

```diff
@@ -1420,11 +1420,14 @@ components:
       required:
         - name
         - riskAnalysisForm
+        - tenantKind
       properties:
         name:
           type: string
         riskAnalysisForm:
           $ref: "#/components/schemas/EServiceRiskAnalysisFormSeed"
+        tenantKind:
```

Comment:
Minor: The `EServiceRiskAnalysisSeed` schema is used by both create and update endpoints for Risk Analysis Templates. However, in the case of the update, providing a tenantKind is strictly necessary?

Reply:
Yes, it is needed because the user could decide to update the risk analysis to another kind